src/app/schemas/sequences.py

```diff
@@ -18,7 +18,7 @@ class SequenceUpdate(BaseModel):
 
 
 class SequenceLabel(BaseModel):
-    is_wildfire: bool
+    is_wildfire: AnnotationType
```

Comment:
So this will automatically do the validation at the API level of the string that is passed in?

Reply:
yes :)